eng/build.ps1

```diff
@@ -51,6 +51,7 @@ param (
   [string]$officialSkipTests = "",
   [string]$officialSourceBranchName = "",
   [string]$officialIbcDrop = "",
+  [string]$officialDevDivDropAccessToken = "",
```

Comment:
>$officialDevDivDropAccessToken [](start = 10, length = 30)

Nit: I'd call this `$officialVisualStudioDropAccessToken` to match the msbuild property name